woocommerce-gateway-ebanx/gateways/class-wc-ebanx-debit-card-gateway.php

```diff
@@ -87,7 +93,7 @@ public function payment_fields()
 	protected function request_data($order)
 	{
 		if (empty($_POST['ebanx_debit_token']) || empty($_POST['ebanx_billing_cvv'])) {
```

Comment:
We're not using superglobals anymore, please replace `empty($_POST[<value>])` with `empty(WC_EBANX_Request::read(<value>))`